doc/build/build.md

```diff
@@ -288,6 +288,18 @@ ctest -R gpu
 Another useful option is `--output-on-failure`, which will print verbose output
 in case a test fails. Full set of options can be found [here](https://cmake.org/cmake/help/latest/manual/ctest.1.html).
 
+@warning
+On Linux, binaries (such as examples, benchdnn, or gtests) may have either a
+`RUNPATH` or `RPATH` entry, depending on the linker version.
+To check, run: `readelf -d <binary> | grep -i path`.
+When using the `/opt/intel/oneapi/setvars.sh` script from the Intel oneAPI toolkit,
+`LD_LIBRARY_PATH` is set to include the oneDNN library path from the installation.
+If the binary contains a `RUNPATH` entry, the dynamic linker will use `LD_LIBRARY_PATH` to
+find the oneDNN library. Make sure the correct oneDNN library is present in
+`LD_LIBRARY_PATH` by setting it explicitly if needed.
+If the binary has an `RPATH` entry, the path in `RPATH` will take precedence
+over `LD_LIBRARY_PATH` when searching for the oneDNN library.
```

Comment:
I'd suggest dropping explaination of `RPATH`/`RUNPATH` mechanics. It should be sufficient to note that setting `LD_LIBRARY_PATH` is required for tests/examples to work properly when `libdnnl.so` is present in the environment.

Reply:
updated